### PR TITLE
meta-kanto layer for Hardknott Yocto image

### PR DESCRIPTION
**Pull request by Mahesha-M**
Described step by step instructions to include meta-kanto for hardknott yocto image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,50 @@
...
 
-# Table of Contents
+1. Clone the poky receipies in separate dictonary: 
+    git clone https://git.yoctoproject.org/poky
 
-1. [Adding the meta-kanto layer to your build](#adding-the-meta-kanto-layer-to-your-build)
-2. [Required image configurations](#required-image-configurations)
-3. [Example image configuration](#example-image-configuration)
+2. Checkout the poky branch that includes the required Go version:
+    git ckecout <required_go_version_poky_branch>
```
Comment (hristobojilov) [inline README.md:43]: Typo in the git command. Should be git checkout

File: README.md [hunk 2]
```diff
@@ -1,44 +1,50 @@
...
 
-## Adding the meta-kanto layer to your build
+3. Replace the Go version in your project build path "poky\meta\recipes-devtools" with
```
Comment (hristobojilov) [inline README.md:45]: Add example what to replace exactly with what maybe be s some linux tools like sed or awk or patch

File: README.md [hunk 3]
```diff
@@ -57,7 +63,54 @@ To do that, add in the configuration file the following line:
...
+## Adding edge containerization to your build
+
+The edge containerization (i.e. Eclipse Kanto container management) serves as a
+specialized component within the Eclipse Kanto and if it is to be included in your
+target Yocto image, you need to enable the required virtualization layer.
+To do that, add in the image configuration file the following line:
+
+    DISTRO_FEATURES:append = " virtualization"
+
+For further fine-tuning and options see the meta-virtualization README.
```
Comment (hristobojilov) [inline README.md:89]: Add also link to this  meta-virtualization README file here

File: README.md [hunk 4]
```diff
@@ -1,44 +1,50 @@
 ![Kanto logo](https://github.com/eclipse-kanto/kanto/raw/main/logo/kanto.svg)
 
-# Eclipse Kanto - Yocto Layer
+# Eclipse Kanto - Essential IoT enablement for edge devices
```
Comment (hristobojilov) [inline README.md:3]: You have to fix github validation workflow for this branch https://github.com/eclipse-kanto/meta-kanto/blob/hardknott/.github/workflows/validation.yml

**Review by hristobojilov — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Mahesha-M) [reply README.md:89]: Added the hardknott meta-virtualization README link.

File: README.md (hunk 1 above)
Comment (Mahesha-M) [reply README.md]: I've mentioned the examples for getting intended Go version from poky yocto branch

File: README.md (hunk 2 above)
Comment (Mahesha-M) [reply README.md]: Added examples

File: README.md (hunk 4 above)
Comment (Mahesha-M) [reply README.md:3]: Fixed the validation script

**Pull request merged**